docs/docs/docs_es/introduction/motivation.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Motivación
+---
+
+Por razones de compatibilidad y simplicidad, es mejor usar las funciones nativas de React para el manejo de estados, en lugar de utilizar estados externos y globales. Pero React tiene algunas limitantes:
+
+- El estado de un componente solo puede ser compartido, si éste estado se mueve hacia el componente padre que se tiene en común, pero este implica tener un árbol de componentes mas grande que renderizar.
```

Comment:
Remove first comma, mas->más. This paragraph could be phrased in a more clear way

Reply:
I tried a different way to create the paragraph.

Reply:
Yup, new version reads much better